File: 백준/Bronze/1267. 핸드폰 요금/핸드폰 요금.cc
#include <iostream>

using namespace std;

int main()
{
    // 통화의 개수 N
    int callCount;

    // 영식 요금제
    int youngsikPee = 0;
    // 민식 요금제
    int minsikPee = 0;

    int callTime;

    cin >> callCount;

    for (int i = 0; i < callCount; i++) {
        cin >> callTime;

        youngsikPee = youngsikPee + (((callTime / 30) + 1) * 10);
        minsikPee = minsikPee + (((callTime / 60) + 1) * 15);
    }

    if (youngsikPee > minsikPee) {
        cout << "M " << minsikPee;
    }
    else if (youngsikPee < minsikPee) {
        cout << "Y " << youngsikPee;
    }
    else {
        cout << "Y M " << youngsikPee;
    }

    return 0;
}

File: 백준/Bronze/1267. 핸드폰 요금/핸드폰 요금.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner s1 = new Scanner(System.in);

        // 통화의 개수 N
        int callCount;

        // 영식 요금제
        int youngsikPee = 0;
        // 민식 요금제
        int minsikPee = 0;

        int callTime;

        callCount = s1.nextInt();

        for (int i = 0; i < callCount; i++) {
            callTime = s1.nextInt();

            youngsikPee = youngsikPee + (((callTime / 30) + 1) * 10);
            minsikPee = minsikPee + (((callTime / 60) + 1) * 15);
        }

        if (youngsikPee > minsikPee) {
            System.out.print("M " + String.valueOf(minsikPee));
        } else if (youngsikPee < minsikPee) {
            System.out.print("Y " + String.valueOf(youngsikPee));
        } else {
            System.out.print("Y M " + String.valueOf(youngsikPee));
        }

        s1.close();

    }
}
